src/components/refactor/dialogs/lines-attach-to-split-lines/lines-attach-to-split-lines-form.js

```diff
@@ -0,0 +1,229 @@
+
+    const lineToAttachTo1Field = (
+        <AutocompleteInput
+            outputTransform={(value) => (value === '' ? null : value)}
+            allowNewValue
+            forcePopupIcon
+            name={LINE_TO_ATTACH_TO_ID_1}
+            label="Line1"
+            options={linesIds?.sort((a, b) => a.localeCompare(b))}
+            size={'small'}
+        />
+    );
+
+    const lineToAttachTo2Field = (
+        <AutocompleteInput
+            allowNewValue
+            forcePopupIcon
+            name={LINE_TO_ATTACH_TO_ID_2}
+            label="Line2"
+            options={linesIds?.sort((a, b) => a.localeCompare(b))}
+            size={'small'}
+        />
+    );
+
+    const attachedLineField = (
+        <AutocompleteInput
+            allowNewValue
+            forcePopupIcon
+            name={ATTACHED_LINE_ID}
+            label="LineAttached"
+            options={linesIds?.sort((a, b) => a.localeCompare(b))}
+            size={'small'}
+        />
+    );
+
+    const voltageLevelIdField = (
+        <AutocompleteInput
+            outputTransform={getObjectId}
+            allowNewValue
+            forcePopupIcon
+            name={VOLTAGE_LEVEL_ID}
+            label="AttachedVoltageLevelId"
+            options={voltageLevelOptions.map((voltageLevel) => voltageLevel.id)}
+            onChangeCallback={resetBusBarSection}
+            size={'small'}
+        />
+    );
+
+    const bbsOrNodeIdField = (
+        <AutocompleteInput
+            allowNewValue
+            forcePopupIcon
+            name={BUS_BAR_SECTION_ID}
+            label="BusBarBus"
+            options={busOrBusbarSectionOptions}
+            inputTransform={(value) => (value === null ? '' : value)}
+            size={'small'}
+        />
+    );
+
+    const newLine1IdField = (
+        <TextInput name={REMPLACING_LINE_ID_1} label={'Line1ID'} />
+    );
+
+    const newLine1NameField = (
+        <TextInput name={REMPLACING_LINE_NAME_1} label={'Line1Name'} />
+    );
+
+    const newLine2IdField = (
+        <TextInput name={REMPLACING_LINE_ID_2} label={'Line2ID'} />
+    );
+
+    const newLine2NameField = (
+        <TextInput name={REMPLACING_LINE_NAME_2} label={'Line2Name'} />
+    );
+
+    return (
+        <>
+            <GridSection title="Line1" />
+            <Grid container spacing={2} alignItems="center">
+                {gridItem(lineToAttachTo1Field, 5)}
+            </Grid>
+            <GridSection title="Line2" />
+            <Grid container spacing={2} alignItems="center">
+                {gridItem(lineToAttachTo2Field, 5)}
+            </Grid>
+            <GridSection title="LineAttached" />
+            <Grid container spacing={2} alignItems="center">
+                {gridItem(attachedLineField, 5)}
+            </Grid>
+            <GridSection title="VoltageLevel" />
+            <Grid container spacing={2}>
+                {gridItem(voltageLevelIdField)}
+                {gridItem(bbsOrNodeIdField)}
+            </Grid>
```

Comment:
I think you could use ConnectivityForm with "withDirectionsInfos" equals to false
It would isolate the "watchVoltageLevelId" to prevent the whole dialog to render